gadgets/mri_core/NoiseAdjustGadget.cpp

```diff
@@ -428,9 +433,19 @@ namespace Gadgetron{
       }
 
       if (noise_decorrelation_calculated_) {
-	//Apply prewhitener
-	hoNDArray<std::complex<float> > tmp(*m2->getObjectPtr());
-	gemm(*m2->getObjectPtr(), tmp, noise_prewhitener_matrixf_);
+          //Apply prewhitener
```

Comment:
Shouldn't this be:

      if ( noise_prewhitener_matrixf_.get_size(0) == m2->getObjectPtr()->get_size(1) ) {
           hoNDArray<std::complex<float> > tmp(*m2->getObjectPtr());
           gemm(*m2->getObjectPtr(), tmp, noise_prewhitener_matrixf_);
      } else {
           if (!pass_nonconformant_data_) {
                 m1->release();
                 return GADGET_FAIL;
           }
      }